metaflow/plugins/argo/argo_workflows_cli.py

```diff
@@ -511,3 +501,81 @@ def _convert_value(param):
         "(run-id *{run_id}*).".format(name=obj.workflow_name, run_id=run_id),
         bold=True,
     )
+
+
+@argo_workflows.command(help="Delete the Argo workflow")
+@click.option(
+    "--authorize",
+    default=None,
+    type=str,
+    help="Authorize the deletion with a production token",
+)
+@click.pass_obj
+def delete(obj, authorize=None):
+    validate_token(obj.workflow_name, obj.token_prefix, obj, authorize, "delete")
+    obj.echo("Deleting workflow *{name}*...".format(name=obj.workflow_name))
+    schedule_deleted = ArgoWorkflows.remove_schedule(obj.workflow_name)
```

Comment:
Might need to account for any `@trigger` or `@trigger_on_finish` resources (Argo Events `Sensor` objects in k8s).  Not sure if this PR merges first or last vs that [PR](https://github.com/Netflix/metaflow/pull/1271).

Reply:
if the other PR gets merged before this, I'll account for the teardown of the `Sensor` as well. Any thoughts on if it can be treated the same way as the cronworkflow deletion, or should errors on sensor deletion be blocking?